src/lib/holocene/table/paginated-table/paginated.svelte

```diff
@@ -83,8 +86,21 @@
     });
   };
 
+  const scrollToHashEvent = async () => {
+    store.jumpToHashPage(hash);
+    await tick();
+    let id = hash.slice(1);
+    const row = document?.querySelector(`[data-eventid="${id}"]`);
+    if (row) {
+      setTimeout(() => {
+        row?.scrollIntoView({ behavior: 'smooth' });
+      }, 500);
+    }
+  };
+
```

Comment:
I'm not sure how I feel about this logic being in a holocene component and enabled by default. Was there a reason you didn't add this to the parent wherever it was needed?

Reply:
Yeah I was on the fence about that too. I could expose jumpToHash to the slot but then you would need to run the scrollToHashEvent function on every row, which doesn't work if the row isn't rendered (i.e. on the same page you are looking at).

I do think we'll want this eventually for other paginated tables but agreed that this isn't ideal in that it's for events only. I added a hashField prop so that it only runs if that's provided and then you can pass whatever data-{hashField} attribute you want to scroll